bin/logstash

```diff
@@ -8,7 +8,19 @@ export ELASTICSEARCH_HOST=$url
 export ELASTICSEARCH_USER=$user
 export ELASTICSEARCH_PASSWORD=$pass
 
-export LS_JAVA_OPTS="$JAVA_OPTS"
+## Allow the ability to verride
+if [ -d "$APP_PATH/config" ] ; then
+  cp $APP_PATH/config/* "${APP_PATH}/logstash/logstash-${LOGSTASH_VERSION}/config"
```

Comment:
Can you document this feature on the documentation (https://doc.scalingo.com/platform/getting-started/getting-started-with-elk#logstash)?

Reply:
Yes!